DependencyInjection/HttplugExtension.php

```diff
@@ -162,6 +162,9 @@ private function configurePluginByName($name, Definition $definition, array $con
                 $definition->replaceArgument(0, new Reference($config['journal']));
                 break;
             case 'logger':
+                if (!$container->has($config['logger'])) {
```

Comment:
this is not working. The ContainerBuilder you get in the extension is not the main one. It is an empty one, and services added in it by the extension are then merged into the main ContainerBuilder.

the reason for that is to forbid extension to change logic based on which services are defined by other bundles (as they might not have been called yet at this point)